curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67f8ef57b9974052e5ef6b30.md

```diff
@@ -0,0 +1,120 @@
+---
+id: 67f8ef57b9974052e5ef6b30
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Use the ternary operator inside the `results` `div` to conditionally display content based on the `results` array. When the array contains items, display each fruit name inside a `div` with a class of `result-item`. When the array is empty, display a message that reads exactly: `No results found`.
```

Comment:
I feel like paragraph elements would be more appropriate here than generic divs. So I would update that here, in the hints, tests and following seed code.